src/WebAPI/WebAPI.csproj

```diff
@@ -36,6 +36,6 @@
 		<RunPostBuildEvent>OnBuildSuccess</RunPostBuildEvent>
 	</PropertyGroup>
 	<Target Name="SwaggerPostBuildClientAndSpec" AfterTargets="PostBuildEvent" Condition=" '$(Configuration)' == 'Debug' ">
```

Comment:
Consider switching from NSwag to built in .NET 9 OpenAPI support. Perhaps create a PBI in the backlog for this.